### PR TITLE
catalog update

### PR DESCRIPTION
**Pull request by zvuki**
`image_url`s & more

### CODE REVIEW

File: swagger/v3-catalog.yaml [hunk 1]
```diff
@@ -87,134 +102,191 @@ paths:
...
+          description: |
+            List of product images and meta
+          schema:
+            $ref: "#/definitions/ProductImageCollectionResponse"
     post:
+      tags:
+        - Catalog
       description: |
-        Creates an image on a product. Publicly accessible URLs and files (`form post`) are valid parameters.
+        Creates an image on a product. Publically accessible URLs and files (form post) are valid parameters.
```
Comment (bookernath) [inline swagger/v3-catalog.yaml]: "publicly" is the preferred spelling.

File: swagger/v3-catalog.yaml [hunk 2]
```diff
@@ -87,134 +102,191 @@ paths:
...
-          $ref: '#/responses/NotFound'
+          description: |
+            The resource was not found
+          schema:
+            $ref: '#/definitions/NotFound'
   /catalog/products/{product_id}/images/{image_id}:
     parameters:
       - name: product_id
         description: |
-          The ID of the `Product` to which the image is being attached.
```
Comment (bookernath) [inline swagger/v3-catalog.yaml]: Previous version was more correct in terms of capitalization and grammar, let's delete this diff

File: swagger/v3-catalog.yaml [hunk 3]
```diff
@@ -87,134 +102,191 @@ paths:
...
+          The id of the `Product` the image is being attached to.
         required: true
         in: path
         type: integer
       - name: image_id
         description: |
-          The ID of the `Image` being operated on.
+          The id of the `Image` the image is being operated on.
```
Comment (bookernath) [inline swagger/v3-catalog.yaml]: Recommend: "The unique ID of the `Image`"

**Comment by bc-sandeep** [comment]
👍 looks good to me, seems all changes from latest vx-swagger are here

**Pull request merged**